Adds new ldap fixed API changes and ldap general docs updates (#2133)
hashicorp/web-unified-docs: content/vault/v2.x (rc)/content/api-docs/secret/ldap.mdx

| `GET`    | `/ldap/static-migration`      |
| `POST`   | `/ldap/static-migration`      |

### Parameters

- `include_role_names` `(boolean: <required>)` – Includes lists of migrated role names and unmigrated role names when reading

>   the migration status. `include_role_names` is required to be set to either true or false and has no default value.

Reviewer: ```suggestion
  the migration status. Vault does not set a default value for `include_role_names`. You must set `include_role_names` explicitly.
```
Style correction: write in active voice

The `GET` endpoint returns statistics about in-progress migration attempts or nothing if no migration is active. Include
`include_role_names` to list all role names that are finished migrating or still need to be migrated. Be aware that this

> can result in a very large response if there are many static roles.

Reviewer: ```suggestion
The `GET` endpoint returns statistics about in-progress migration attempts. If
Vault does not have an active migration, the endpoint returns nothing. To include
completed and pending migrations in the result, set the `include_role_names`
parameter. Be aware that including all migrations can result in a very large
response if you have many static roles.
```
Style correction: write in active voice
